Mage/src/main/java/mage/filter/predicate/permanent/ControllerPredicate.java

```diff
@@ -69,6 +69,10 @@ public boolean apply(ObjectPlayer<Controllable> input, Game game) {
                     return true;
                 }
                 break;
+            case ACTIVE:
+                if (object.getControllerId().equals(game.getActivePlayerId())) {
+                    return true;
```

Comment:
Ok never mind it is working 100% correctly now. I am no longer able to select creatures controlled by nettling imps controller. I think when i previously tested the revisions commited that netbeans didn't compile something.